aquadoggo/src/bus.rs

```diff
@@ -1,17 +1,15 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
-use p2panda_rs::entry::Entry;
-use p2panda_rs::operation::Operation;
+use p2panda_rs::operation::OperationId;
 
 use crate::manager::Sender;
 
 /// Sender for cross-service communication bus.
 pub type ServiceSender = Sender<ServiceMessage>;
 
 /// Messages which can be sent on the communication bus.
-#[derive(Clone)]
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub enum ServiceMessage {
-    /// New `Entry` with an `Operation` payload arrived at the node.
-    #[allow(dead_code)]
-    NewEntryAndOperation(Entry, Operation),
+    /// A new operation arrived at the node.
+    NewOperation(OperationId),
 }
```

Comment:
Just an observation, but does the service manager never deal with entries then? Only operations?

Would there be any merit in someday having a service dedicated to storing entries and then dispatching following tasks? We might then be able to consolidate the logic for storing both entries arriving via replication and via publish entry. Haven't thought that through very far so it may not make sense.

Reply:
Yeah, this enum is the list of all things which can be sent to all services, you're right then, with this change it will for now only be `OperationId`, but as soon as we see need for something else, we can readjust or simple just add new message types :+1:

Reply:
As a side note: Not all services need to process all incoming messages, so there might be different messages different services are interested in / or not. Later (piet suggested that), we can have even dedicated channels (or even an actor model) for these things, but I think thats premature optimization at this stage (https://github.com/p2panda/aquadoggo/issues/93)